src/ducks/programs.js

```diff
@@ -137,7 +162,7 @@ const gorongosa = {
 
 const kenya = {
   staging: {
-    id: '5',
+    id: '6',
```

Comment:
Wait, you're right, Kenya IS Program 6 on staging but Program 5 on production. Goshdangit, I think I remember that this might have been my fault - I could create a new Program _but not edit it,_ so when I found a mistake, I think I had to delete Program 5 on staging then create a new Program 6 with the correct details this time. This lack of synchronisation is going to haunt my goshdang dreams. 🤨